### PR TITLE
VT: improving bill action classification

### PR DESCRIPTION
**Pull request by sruthivedantham**
Improvements associated with [OS-783](https://civic-eagle.atlassian.net/browse/OS-783). 

These changes update the VT scraper to:

- match bill action phrases to existing [OS classifications](https://github.com/openstates/openstates-core/blob/5b16776b1882da925e8e8d5c0a07160a7d649c69/openstates/data/common.py#L87) at a higher rate than before
- collect bill action phrases and their respective classifications in an organized way
- perform bill action classification in a separate function (outside of the main bill scraping function)

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 default_language_version:
-    python: python3.9
```
Comment (johnseekins) [inline .pre-commit-config.yaml:2]: This actually does matter. Probably don't change this.

File: scrapers/vt/bills.py [hunk 2]
```diff
@@ -9,6 +9,40 @@
...
+        "vetoed": ["executive-veto"],
+        "governor allowed to become law": ["became-law"],
+        "allowed to go into effect without the signature of the governor": [
+            "became-law"
+        ],
+        "proposal of amendment": ["amendment-introduction"],
+    }
+
+    def categorize_actions(self, action_string):
+        for action_key in list(self._actions):
```
Comment (johnseekins) [inline scrapers/vt/bills.py:40]: `for action_key in self._actions.keys()`?

File: scrapers/vt/bills.py [hunk 3]
```diff
@@ -198,24 +232,18 @@ def scrape(self, session=None):
...
+                )
+
+                # If both an action type and actor were assigned
+                if action_classification:
+                    action_type = action_classification[0]
+
+                    if len(action_classification) > 1:
+                        actor = action_classification[1]
+
+                # More action categorization (these were the tricky ones)
```
Comment (johnseekins) [inline scrapers/vt/bills.py:246]: Since these are tricky, is it worth adding some examples in comments?

File: scrapers/vt/bills.py (hunk 2 above)
Comment (sruthivedantham) [reply scrapers/vt/bills.py:40]: good point! will change

File: scrapers/vt/bills.py [hunk 4]
```diff
@@ -9,6 +9,41 @@
 
 
 class VTBillScraper(Scraper, LXMLMixin):
+
+    # Dictionary matching bill action phrases to classifications
```
Comment (johnseekins) [inline scrapers/vt/bills.py:13]: Is there a location that our final classification list exists? Is it worth linking to that here?

Comment (sruthivedantham) [reply scrapers/vt/bills.py:13]: yes there is! I will link

**Pull request merged**